### DISCUSSION TITLE
Element isnt found when opening an url in a new window.

### DISCUSSION THREAD

**Discussion by RickVK04** [seleniumbase/SeleniumBase#2279] [Q&A] [answered]
Hi there,

Ive made some code that loads up a website and puts some random words into a text area. It works perfectly fine when loading the website with open(url) etc. However, when I open a new window, and switch to that window and then open the same url using the exact same code, the text area isnt found. Doesnt matter if I add driver.sleep, increase the timeout etc. I dont understand how it works perfectly fine but then when using it in a new window it doesnt. Anyone encountered the same issue?

**Comment by mdmintz** [comment, marked as answer]
Hello. Not sure what you're doing, (since you haven't provided any code to work with), but you should try this example test that works with multiple windows/tabs, and switching between them: [SeleniumBase/examples/test_window_switching.py](https://github.com/seleniumbase/SeleniumBase/blob/master/examples/test_window_switching.py)

**Reply by RickVK04** [reply]
I have managed to solve the issue. The code I was talking about was `sb.driver.type('#input', "blablabla")`. It did find #input when I just used `sb.open(url)` but when I first did `sb.open_new_window()`, switched to that window and then did `sb.open(url)` it couldn't find the #input. I fixed this by changing all my clicks I did to get to the #input from `uc_click` to a normal `sb.click()`. No clue why this then does allow the #input to be found, maybe the site protects the input or something.
